src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -33,9 +34,21 @@
 public class Datastore {
 
   private DatastoreService datastore;
+  private List<Laptop> laptopList; 
 
+  //in future, change to generic type to go more than laptops 
   public Datastore() {
     datastore = DatastoreServiceFactory.getDatastoreService();
+    laptopList= new ArrayList<>();
+	 laptopList.add(new Laptop("Dell","Black","Windows",15,800.00));
```

Comment:
Nit: Fix whitespace / indentation. The Google style guide recommends indenting with spaces: https://google.github.io/styleguide/javaguide.html

Reply:
Fixed!